cores/esp8266/Schedule.cpp

```diff
@@ -33,38 +39,33 @@ static scheduled_fn_t* get_fn_unsafe()
         result = sUnused;
         sUnused = sUnused->mNext;
         result->mNext = nullptr;
-        result->callNow.reset(esp8266::polledTimeout::periodicFastUs::alwaysExpired);
     }
     // if no unused items, and count not too high, allocate a new one
     else if (sCount < SCHEDULED_FN_MAX_COUNT)
     {
         result = new scheduled_fn_t;
```

Comment:
You may want to replace `new` with `malloc` only because `new` could throw an exception and this is being executed in an interrupts and it'll just crash at that point, I think.